make_dataset.py

```diff
@@ -375,7 +380,24 @@ def parse_args(args_input=None):
     if os.path.exists(ARGS.output_dir):
         if ARGS.verbose:
             print(f"Clearing stale data from {ARGS.output_dir}.")
-        shutil.rmtree(ARGS.output_dir)
+
+        response = None
+        if not ARGS.no_prompt:
+            response = input(f"Delete output_dir ({ARGS.output_dir})? [y/N]: ")
+
+        if (ARGS.no_prompt) or response in ["y", "ye", "yes"]:
+            try:
+                shutil.rmtree(ARGS.output_dir)
+            except FileNotFoundError:
```

Comment:
I think this (and above) should be Exception still. rmtree might also raise OSError or others (bad file permissions, missing sym-link, ...)

Reply:
ok, deal!